<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VEMainWindow</class>
 <widget class="QMainWindow" name="VEMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>836</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Video Editor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../VideoEditor.qrc">
    <normaloff>:/main/VideoEditor.ico</normaloff>:/main/VideoEditor.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="VideoPlayer" name="videoPlayer" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="VideoSlider" name="videoPosSlider">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="btnPlayPause">
            <property name="text">
             <string>Play</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btnMarkIn">
            <property name="text">
             <string>Mark In</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnMarkOut">
            <property name="text">
             <string>Mark Out</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnResetMarks">
            <property name="text">
             <string>Reset Marks</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QPushButton" name="btnOpenVideo">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>250</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>&amp;Open video</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbxPresets">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>250</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="grpScaling">
          <property name="title">
           <string>Scaling</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QSpinBox" name="valScaleWidth">
               <property name="minimum">
                <number>-1</number>
               </property>
               <property name="maximum">
                <number>7680</number>
               </property>
               <property name="value">
                <number>-1</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>X</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="valScaleHeight">
               <property name="minimum">
                <number>-1</number>
               </property>
               <property name="maximum">
                <number>4320</number>
               </property>
               <property name="value">
                <number>720</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Filtering</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cbxScalingFlags">
             <property name="currentIndex">
              <number>4</number>
             </property>
             <item>
              <property name="text">
               <string>Fast bilinear</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Bilinear</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Bicubic</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Averaging area</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Lanczos</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnConvert">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>250</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>&amp;Convert</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnSettings">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>250</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>&amp;Settings</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnExit">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>250</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>E&amp;xit</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VideoPlayer</class>
   <extends>QWidget</extends>
   <header>VideoPlayer.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>VideoSlider</class>
   <extends>QSlider</extends>
   <header>VideoSlider.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../VideoEditor.qrc"/>
 </resources>
 <connections/>
</ui>
